api-reference/20 Data Visualization Widgets/dxChart/1 Configuration/commonAxisSettings/label/position.md

```diff
@@ -6,7 +6,11 @@ default: 'outside'
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Specifies a label position relative to the chart or its axis.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+
+Use one of the following enums to specify this option when the widget is used as an ASP.NET MVC 5 Control or a DevExtreme-Based ASP.NET Core Control:   
```

Comment:
- We can drop the mention of ASP.NET enums.
- Illustrations of each position would be useful.